async-nats/examples/sub.rs

```diff
@@ -1,9 +1,8 @@
 use futures::stream::StreamExt;
-use std::error::Error;
 use std::time::Instant;
 
 #[tokio::main]
-async fn main() -> Result<(), Box<dyn Error>> {
+async fn main() -> Result<(), async_nats::Error> {
```

Comment:
I'd leave these explicit? since they're the absolute top-most level

```suggestion
async fn main() -> Result<(), async_nats::Error> {
```